server/views/pages/update-goal.njk

```diff
@@ -0,0 +1,168 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "../components/summary-card/goal-summary-card.njk" import goalSummaryCard %}
+
+{% set locale = interpolate(locale, {
+    subject: data.popData,
+    goal: {
+        targetDate: data.goal.targetDate | formatSimpleDate
+    }
+}) %}
+
+{% set items = [
+    {
+        value: "NOT_STARTED",
+        text: locale.common.goalSummaryCard.stepsList.status.notStarted
+    },
+    {
+        value: "IN_PROGRESS",
+        text: locale.common.goalSummaryCard.stepsList.status.inProgress
+    },
+    {
+        value: "CANNOT_BE_DONE_YET",
+        text: locale.common.goalSummaryCard.stepsList.status.cannotBeDoneYet
+    },
+    {
+        value: "NO_LONGER_NEEDED",
+        text: locale.common.goalSummaryCard.stepsList.status.noLongerNeeded
+    },
+    {
+        value: "COMPLETED",
+        text: locale.common.goalSummaryCard.stepsList.status.completed
+    }
+]
+%}
+
+{% block pageTitle %}
+    {% if not errors %}
+        {{ locale.page.title }}
+    {% else %}
+        {{ locale.page.errorTitle }}
+    {% endif %}
+{% endblock %}
+
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block head %}
+    {{ super() }}
+    <style nonce="{{ cspNonce }}">
+        .flex-width {
+            width: 66%
+        }
+        .govuk-select {
+            min-width: auto
+        }
+    </style>
+{% endblock %}
+
+{% block content %}
+
+{{ govukBackLink({
+  text: locale.common.backLink.text,
+  href: "/plan"
+}) }}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+    <span class="govuk-caption-l flex-width">{{ data.mainAreaOfNeed.name }}
+    {% if data.relatedAreasOfNeed | length %}
+        (and {{ data.relatedAreasOfNeed | join(", ") }}){% endif %}</span>
```

Comment:
frequently these needs to be sorted and lowercased (they are capitalised in the API DB) so `data.relatedAreasOfNeed | sort | join(', ') | lower`

Reply:
Amended in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/170/commits/9d4fecdaf974272a38eede3de4594e9ee9d3df75